docker/events.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import meetup.api
+from slacker import Slacker
+
+
+def getSecret(service, dirPath=os.curdir):
+    fname = '.secrets.json'
+    if not os.path.isfile(os.path.join(dirPath, fname)):
+        exit
+    with open(fname) as data:
+        data_json = json.loads(data.read())
+    if service == 'Meetup':
+        return data_json['Meetup']['api_key']
+    elif service == 'Slack':
+        return data_json['Slack']['OAuth_Access_Token']
+
+
+def getEvents(status):
+    client = meetup.api.Client(getSecret(service='Meetup'))
+    events = client.GetEvents({'group_urlname': 'DevOps-Loft',
+                              'status': status})
+    events = json.dumps(events.__dict__)
+    events = json.loads(events)
+    return events
+
+
+def putPost(service='Slack', channel='sandbox', message='Hello World',
+            event=None):
+    try:
```

Comment:
Suggest throwing an error if a value other than 'Slack' is passed or rename the method and remove the variable, if someone will use the function in the future, he will have no indication why it doesn't work

Reply:
+1